src/pages/my_registrations/MyRegistrations.tsx

```diff
@@ -1,67 +1,64 @@
   const { t } = useTranslation();
-  const [selectedTab, setSelectedTab] = useState(Tab.Unpublished);
-  const [myRegistrationsResponse, isLoading, refetchRegistrations] = useFetch<MyRegistrationsResponse>({
-    url: PublicationsApiPath.RegistrationsByOwner,
-    errorMessage: t('feedback.error.get_registrations'),
-    withAuthentication: true,
+
+  const registrationsQuery = useQuery({
+    queryKey: ['by-owner'],
+    queryFn: fetchRegistrationsByOwner,
+    meta: { errorMessage: t('feedback.error.search') },
   });
-  const registrations = myRegistrationsResponse?.publications ?? [];
 
-  const unpublishedRegistrations = registrations
-    .filter(({ status }) => status === RegistrationStatus.Draft)
-    .sort((a, b) => new Date(b.createdDate).getTime() - new Date(a.createdDate).getTime());
+  const registrations = registrationsQuery.data?.publications ?? [];
 
-  const publishedRegistrations = registrations
-    .filter(({ status }) => status === RegistrationStatus.Published || status === RegistrationStatus.PublishedMetadata)
-    .sort((a, b) => new Date(b.createdDate).getTime() - new Date(a.createdDate).getTime());
+  const filteredRegistrations = registrations
+    .filter(
+      ({ status }) =>
+        (status === RegistrationStatus.Draft && selectedUnpublished) ||
+        ((status === RegistrationStatus.Published || status === RegistrationStatus.PublishedMetadata) &&
+          selectedPublished)
+    )
+    .sort((a, b) => {
+      if (a.status === RegistrationStatus.Draft || b.status !== RegistrationStatus.Draft) {
+        return -1;
+      } else if (b.status === RegistrationStatus.Draft) {
+        return 1;
+      }
+      return 0;
+    });
 
   return (
     <>
       <Helmet>
         <title>{t('common.registrations')}</title>
       </Helmet>
-      <BackgroundDiv>
-        <Box sx={{ display: 'flex', justifyContent: 'center', marginBottom: '1rem' }}>
-          <TabButton
-            data-testid="unpublished-button"
-            onClick={() => setSelectedTab(Tab.Unpublished)}
-            isSelected={selectedTab === Tab.Unpublished}>
-            {t('my_page.registrations.unpublished_registrations')} ({unpublishedRegistrations.length})
-          </TabButton>
-          <TabButton
-            data-testid="published-button"
-            onClick={() => setSelectedTab(Tab.Published)}
-            isSelected={selectedTab === Tab.Published}>
-            {t('my_page.registrations.published_registrations')} ({publishedRegistrations.length})
-          </TabButton>
-        </Box>
-        {isLoading ? (
+      <>
+        {registrationsQuery.isLoading ? (
           <ListSkeleton minWidth={100} maxWidth={100} height={100} />
         ) : (
-          <MyRegistrationsList
-            registrations={selectedTab === Tab.Unpublished ? unpublishedRegistrations : publishedRegistrations}
-            refetchRegistrations={refetchRegistrations}
-          />
+          <Box>
```

Comment:
når vi trenger en tom container er det bedre med en native div enn en MUI-Box
```suggestion
          <>
```

Reply:
EDIT: Om `<div>` settes i stedet for tom tag noen linjer over her, så kan vi sikkert bruke tom tag her i stedet (?)